src/resources/Sources/Sources.ts

```diff
@@ -125,10 +123,14 @@ export default class Sources extends Resource {
         return this.api.get<RawSourceConfig>(`${Sources.baseUrl}/${sourceId}/raw`);
     }
 
+    getBasicSources() {
+        return this.api.get<BasicSourceModel[]>(`${Sources.baseUrl}/basic/all`);
```

Comment:
This call doesn't seem to exist (be public) in prod right now

Reply:
What is the purpose? We would essentially be documenting it by adding it to the p-c so the same purpose would apply here no?

Reply:
Hello! We don't own this endpoint. We needed a new one for our use case and the source team implemented it, so the fact that it should documented in the p-c or not is not our decision (and our concern either), but you can discuss this matter with the source team if you think this is an issue. 
cc. @jmorissette

Reply:
Basically my concern is that we want only public, documented, and stable stuff in the platform client. This is an open source repo after all and it is used externally by clients directly. 

If it isn't the case, for the endpoint in question, it can still be used by leveraging an [experimental client instance](https://github.com/coveo/platform-client#extending-the-client) in your project until the endpoint is public, documented and stable.

Reply:
If the intent is to never make it public, then I guess it should not live here. Is there something I'm missing?

Reply:
I'm with Gael on this. If we don't want to document this call in swagger because we don't want clients to use it then we shouldn't put it in a public repository.

If you need help to add it to the experimental client instance we're happy to help

Reply:
Thanks for the input! I understand the points from both sides. I will use an experimental client for this purpose. Thanks for offering help @GermainBergeron but I should be able to handle it.